### DISCUSSION TITLE
关于短视频的CQ码, 官方文档和docs里的文档不同

### DISCUSSION THREAD

**Discussion by bin-tenkuu** [Mrs4s/go-cqhttp#786] [Q&A] [answered]
关于短视频的CQ码, 官方文档和docs里的文档不同.
在[额外文档](https://github.com/Mrs4s/go-cqhttp/blob/master/docs/cqhttp.md#%E7%9F%AD%E8%A7%86%E9%A2%91%E6%B6%88%E6%81%AF)中是这么写的:

|参数名 | 类型 | 说明|
|-- | -- | -- |
|file | string | 支持http和file发送|
|cover | string | 视频封面，支持http，file和base64发送，格式必须为jpg|
|c | `2` `3` | 通过网络下载视频时的线程数, 默认单线程. (在资源不支持并发时会自动处理)|

在[go-cqhttp 帮助中心 - CQ code](https://ishkong.github.io/go-cqhttp-docs/cqcode/#%E7%9F%AD%E8%A7%86%E9%A2%91)中是这么写的:

|参数名 | 收 | 发 | 可能的值 | 说明|
|-- | -- | -- | -- | -- |
|file | ✓ | ✓[1] | - | 视频文件名|
|url | ✓ |   | - | 视频 URL|
|cache |   | ✓ | `0` `1` | 只在通过网络 URL 发送时有效, 表示是否使用已缓存的文件, 默认 1|
|proxy |   | ✓ | `0` `1` | 只在通过网络 URL 发送时有效, 表示是否通过代理下载文件（需通过环境变量或配置文件配置代理）, 默认 1|
|timeout |   | ✓ | - | 只在通过网络 URL 发送时有效, 单位秒, 表示下载网络文件的超时时间 , 默认不超时|

我应该信哪一个

还是说一个是用于接收, 一个用于发送

写 CQ码生成类 的时候发现文档不一致, 来问问

**Comment by wdvxdr1123** [comment, marked as answer]
这个CQ码，docs文件夹中的文档是正确的

**Reply by bin-tenkuu** [reply]
好, 感谢告知
